src/components/MenuBar/ControlCenter/index.tsx

```diff
@@ -15,8 +15,8 @@ type ControlCenterProps = {
   togglePlay?: () => void;
 };
 
-const ControlCenter = ({ isPlayingMusic, togglePlay }: ControlCenterProps) => {
-  const { sound, brightness } = useContext(SystemStateContext);
+const ControlCenter = ({ togglePlay }: ControlCenterProps) => {
+  const { sound, brightness, isPlaying } = useContext(SystemStateContext);
   const dispatch = useContext(SystemReducerContext);
 
   if (!dispatch) throw new Error('dispatch is null');
```

Comment:
```suggestion
const ControlCenter = () => {
  const { sound, brightness } = useContext(SystemStateContext);
  const dispatch = useContext(SystemReducerContext);

  if (!dispatch) throw new Error('dispatch is null');

  const { isPlaying, togglePlay } = useAudio({ src: MusicSrc });
```

단순히 `isPlaying` 상태가 변하지 않는 문제라면 MenuBar에서 상태를 관리 해야만 하는 이유는 없어보이기 때문에, `ControlCenter` 컴포넌트 내부에서 `useAudio`를 사용하는 것으로 문제를 해결할 수 있습니다! 😀

Reply:
@jaeung-E 아 맞네요! 애초에`ControlCenter` 컴포넌트가 언마운트 됐을 때 `isPlaying` 상태를 유지하기 위해서 `MenuBar` 컴포넌트로 올렸었는데 리팩터링 과정에서 이를 놓친 것 같아요 😅 

꼼꼼한 리뷰 감사드립니다 👍

Reply:
@jaeung-E 말씀대로 `useAudio` 훅을 `ControlCenter` 컴포넌트 내부로 이동했는데 다른 문제가 있어요. `ControlCenter` 컴포넌트가 언마운트 되고, 다시 마운트 됐을 때 새로운 Audio 객체를 만들고 있어요. (리모트 브랜치에서 증상 재현 가능합니다!)

이를 해결하려면,
* `audio` 요소를 `MenuBar` 컴포넌트에 위치시키거나
* `useAudio` 훅을 다시 `MenuBar` 컴포넌트로 올리거나
* 전역으로 사용되는 Audio 객체를 만드는 방법
이 있을 것 같아요.

다른 앱에서도 음악을 제어할 수 있는 기능을 제공한다면 전역으로 관리하는게 맞을 거 같은데, 지금 상황에서는 상단으로 끌어올리는게 적절하지 않을까 싶어요. 재웅님 생각은 어떠신가요? 🤔

Reply:
@minjongbaek 맞네요! `useAudio` hook이 오디오 객체를 생성하기 때문에 문제가 발생하네요..😅

그렇다면 `useAudio` 를 리팩터링 해서 저번 pr에서 말씀드렸던 대로 오디오 관련 `ControlCenterItem`에 <audio> 요소를 추가하고, ref 객체를 이용해서 오디오를 제어하게 되면 시멘틱 구조도 유지하면서, 오디오 객체가 여러개 생기는 문제도 해결할 수 있을 것 같습니다! 😀

Reply:
`ControlCenterItem` 내부에 `<audio>` 요소를 위치시킨다면 `ControlCenter` 컴포넌트가 언마운트 됐을 때 노래 재생이 끊기는 문제가 있을 것 같아요. 언마운트 되지 않는 `MenuBar` 혹은 `App` 컴포넌트에 위치해야 할 것 같다고 생각해요 🙄

Reply:
@jaeung-E `ControlCenterItem` 내부에 `<audio>` 요소를 위치시킨다면 `ControlCenter` 컴포넌트가 언마운트 됐을 때 노래 재생이 끊기는 문제가 있을 것 같아요. 언마운트 되지 않는 `MenuBar` 혹은 `App` 컴포넌트에 위치해야 할 것 같다고 생각해요 🙄

Reply:
@minjongbaek `unmount`를 다시 확인해보니 실제로 엘리먼트를 제거하고 있었네요..😫 이러면 제안하신 대로 `useAudio`를 다시 상위 컴포넌트로 옮겨야 할 것 같습니다!